highfidelity/hifi-content
# Updated Portal.js

This updates portal.js to the one used currently in world while also addressing Bug 3374, regarding clicking back on GoTo and being teleported again (as the teleporter was the last location the avatar was upon departure).

Testing conditions:
1. Set up teleporter as usual (JSON area should contain domain name of destination)
2. Install script
3. Walk into teleporter. Teleporter should take user to desination
4. Open up GoTo.
5. Click the back button. User should return to previous area inside teleporter.
6. With current update from this pull, user should **NOT** be teleported. Original verison would teleport user back to destination.
